### ISSUE TITLE
PEAR FunctionCallSignature sniff forbids comments after opening parenthesis of a multiline call

### ISSUE THREAD

**Issue by guillaume-d** [squizlabs/PHP_CodeSniffer#768]
The ContentAfterOpenBracket error is triggered.
Note that comments after the arguments are correctly allowed.
All non-doc comment syntaxes (`//`, `/*`, `#`) are affected.

**Comment by aik099** [comment]
Please provide code sample and indicate line at which error is triggered.

**Comment by guillaume-d** [comment]
The following will trigger the error at line 3 and 7:
```php
<?php

file(#
    $s
);

$o->toSring(//
);
```

**Comment by gsherwood** [comment]
Marking as a bug because it affects the PSR2 standard, which ignores comments, and the PEAR standard isn't that clear.
